junit-jupiter-params/src/test/java/org/junit/jupiter/params/ParameterizedTestIntegrationTests.java

```diff
@@ -164,11 +158,25 @@ void executesWithAggregator() {
 					event(test(), displayName("[2] ef, gh"), finishedWithFailure(message("concatenation: efgh"))));
 	}
 
+	@Test
+	void executesWithIgnoreTrailingAndLeadingSetToFalse() {
+		var results = execute("testCsvSourceWithIgnoreTrailingAndLeadingWhitespaceSetToFalse", String.class);
+		results.allEvents().assertThatEvents()
+				.haveAtLeast(1,
+						event(test(), finishedWithFailure(instanceOf(AssertionError.class))));
+	}
+	@Test
+	void executesWithIgnoreTrailingAndLeadingSetToTrue() {
+		var results = execute("testCsvSourceWithIgnoreTrailingAndLeadingWhitespaceSetToTrue", String.class);
+		results.allEvents().assertThatEvents()
+				.haveAtLeast(1,
+						event(test(), finishedWithFailure(instanceOf(AssertionError.class))));
+	}
 	@Test
 	void failsContainerOnEmptyName() {
 		var results = execute("testWithEmptyName", String.class);
 		results.allEvents().assertThatEvents() //
-				.haveExactly(1, event(container(), displayName("testWithEmptyName(String)"), //
+				.haveAtLeast(1, event(container(), displayName("testWithEmptyName(String)"), //
```

Comment:
Why do we need to modify this test?

Reply:
my bad, will fix. i think this was not intentional

Reply:
done, please check the latest commit.